### PR TITLE
add mobile version

### PR DESCRIPTION
**Pull request by leminam**

### CODE REVIEW

File: src/modules/portfolio/.fuse_hidden0000009000000001 [hunk 1]
```diff
@@ -0,0 +1,90 @@
+.portfolio {
```
Comment (lemind) [inline src/modules/portfolio/.fuse_hidden0000009000000001:1]: a strange file

Comment (lemind) [reply src/modules/portfolio/.fuse_hidden0000009000000001:1]: name I mean

File: src/modules/portfolio/portfolio-item.hbs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      <span class="portfolio__devices-item portfolio__devices-item_space">
+        <a href="#" class="portfolio__devices-link">Tablet</a>
+      </span>
+      <span class="portfolio__devices-item">
+        <a href="#" class="portfolio__devices-link">Mobile</a>
+      </span>
+    </div>
+    <div class="portfolio__devices-description">{{ devices_description }}</div>
+    <div class="portfolio__devices-image">
+      <img src="{{ screenshot }}" class="portfolio__devices-screenshot">
```
Comment (lemind) [inline src/modules/portfolio/portfolio-item.hbs:24]: is it same img for  all of this?

File: src/modules/portfolio/.fuse_hidden0000009000000001 (hunk 1 above)
Comment (leminam) [reply src/modules/portfolio/.fuse_hidden0000009000000001:1]: I don't know what it is. Probably css files doubled. What I need to do? Just delete on my computer?

File: src/modules/portfolio/portfolio-item.hbs (hunk 2 above)
Comment (leminam) [reply src/modules/portfolio/portfolio-item.hbs:24]: it is the same in this file, but different in data.js
screenshot: "static/img/screenshot_1.png",
screenshot: "static/img/screenshot_2.png",

File: src/modules/portfolio/.fuse_hidden0000009000000001 (hunk 1 above)
Comment (lemind) [reply src/modules/portfolio/.fuse_hidden0000009000000001:1]: be careful. Yes jsut delete that

File: src/modules/portfolio/portfolio-item.hbs (hunk 2 above)
Comment (lemind) [reply src/modules/portfolio/portfolio-item.hbs:24]: ok

File: src/modules/portfolio/.fuse_hidden0000009000000001 (hunk 1 above)
Comment (leminam) [reply src/modules/portfolio/.fuse_hidden0000009000000001:1]: Just delete like usual files? Or i need to use git rm?

**Pull request merged**